### PR TITLE
add CloudProvider field for customer statement

### PR DESCRIPTION
**Pull request by mboret**

### CODE REVIEW

**Review by Shr3ps — APPROVED** [review]

**Comment by Issif** [comment]
The SDK is still alive 👍

File: customer_statements_test.go [hunk 1]
```diff
@@ -24,6 +25,7 @@ var defaultBillingArtifacts = BillingArtifacts{
 	BillingArtifacts: []BillingArtifact{
 		{
 			CustomerID:    1234567890,
+	                CloudProvider: "AWS",
```
Comment (Issif) [inline customer_statements_test.go:28]:
```suggestion
			CloudProvider: "AWS",
```

File: customer_statements_test.go [hunk 2]
```diff
@@ -35,6 +37,7 @@ var defaultBillingArtifacts = BillingArtifacts{
 		},
 		{
 			CustomerID:    98765433210,
+	                CloudProvider: "AWS",
```
Comment (Issif) [inline customer_statements_test.go]:
```suggestion
			CloudProvider: "AWS",
```

**Review by Issif — CHANGES_REQUESTED** [review]

**Review by pdecat — CHANGES_REQUESTED** [review]
Replace spaces by tabs as suggested by @Issif and you're good to go :)

**Comment by mboret** [comment]
@pdecat Done

**Review by pdecat — APPROVED** [review]
LGTM!

**Comment by mboret** [comment]
@Shr3ps merge please...

**Review by xp-1000 — APPROVED** [review]

**Pull request merged**